depth.py

```diff
@@ -1,28 +1,90 @@
+import argparse
```

Comment:
At the very top of the file, maybe it would be useful to have a docstring with a quick reminder of the intent of this file? Something like:

```py
"""A node depth computation for `.og` files implemented using odgi's
Python bindings. While this implementation reuses odgi's data structures, it
does not reuse its node depth computation algorithm and instead implements
it from scratch.
"""
```